lib/reporting/proofing_rate_report.rb

```diff
@@ -8,7 +8,7 @@ module Reporting
   class ProofingRateReport
     DATE_INTERVALS = [30, 60, 90].freeze
 
-    attr_reader :end_date, :wait_duration
+    attr_reader :end_date, :wait_duration, :by_month
```

Comment:
this can be removed now right?
```suggestion
    attr_reader :end_date, :wait_duration
```